changelog.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.7.0~rc4
+
+* Update v2 deprecation warning date from February to March
```

Comment:
For reference, https://github.com/freedomofpress/securedrop/pull/5734. We'll add that in the final changelog

Reply:
good catch

Reply:
for the final release, i'll have to move these bullets (or merge them with existing bullets) to the 1.7.0 changelog section so we'll come across this again